shell/config/product/explorer.js

```diff
@@ -198,20 +204,47 @@ export function init(store) {
   configureType(MANAGEMENT.PSA, { localOnly: true });
 
   headers(PV, [STATE, NAME_COL, RECLAIM_POLICY, PERSISTENT_VOLUME_CLAIM, PERSISTENT_VOLUME_SOURCE, PV_REASON, AGE]);
-  headers(CONFIG_MAP, [NAME_COL, NAMESPACE_COL, KEYS, AGE]);
+
+  headers(CONFIG_MAP,
+    [NAME_COL, NAMESPACE_COL, KEYS, AGE],
+    [
+      STEVE_NAME_COL,
+      STEVE_NAMESPACE_COL, {
+        ...KEYS,
+        sort:   false,
+        search: false,
+      },
+      STEVE_AGE_COL
+    ]
+  );
+
+  const secretData = {
+    name:      'data',
+    labelKey:  'tableHeaders.data',
+    value:     'dataPreview',
+    formatter: 'SecretData'
+  };
```

Comment:
Do we want this (along with the other types defined in this file) to be with table-headers?